app/javascript/components/explore/StudyGeneField.jsx

```diff
@@ -59,11 +60,15 @@ export default function StudyGeneField({ genes, searchGenes, allGenes, speciesLi
       setShowNotPresentGeneChoice(true)
     } else if (newGeneArray && newGeneArray.length) {
       const genesToSearch = newGeneArray.map(g => g.value)
-      if (event) { // this was not a 'clear'
-        const trigger = event.type // 'click' or 'submit'
-        logStudyGeneSearch(genesToSearch, trigger, speciesList)
+      if (genesToSearch.length > window.MAX_GENE_SEARCH) {
+        setShowTooManyGenesModal(true)
```

Comment:
Let's log these, to let us keep assessing recent impact.  Something like:

```suggestion
        log('search-too-many-genes', {numGenes: genesToSearch.length})
        setShowTooManyGenesModal(true)
```